code/controllers/subsystem/air.dm

```diff
@@ -553,16 +553,21 @@ SUBSYSTEM_DEF(air)
 	// Now we're gonna compare for differences
 	// Taking advantage of current cycle being set to negative before this run to do A->B B->A prevention
 	for(var/turf/open/potential_diff as anything in difference_check)
-		potential_diff.current_cycle = 0
+		// I can't use 0 here, so we're gonna do this instead. If it ever breaks I'll eat my shoe
+		potential_diff.current_cycle = -INFINITE
```

Comment:
y not just use -1

Reply:
because I use -1 to make building atmos_adjacent_turfs faster maybe 20 lines up